src/webots/nodes/utils/WbNodeUtilities.cpp

```diff
@@ -929,12 +929,19 @@ bool WbNodeUtilities::isInBoundingObject(const WbNode *node) {
   const WbNode *const p = node->parentNode();
   if (p) {
     const WbSolid *const s = dynamic_cast<const WbSolid *>(p);
+    const WbFluid *const f = dynamic_cast<const WbFluid *>(p);
```

Comment:
Probably we can also improve the code by using directly `WbMatter` instead of both `WbSolid` and `WbFluid`.

Reply:
Good point, it worked locally. Just waiting for the CI.
Also had to sync with develop because of a conflict.